pkg/deploy/assets/env-development.json

```diff
@@ -93,13 +86,34 @@
             "location": "[resourceGroup().location]",
             "apiVersion": "2020-08-01"
         },
+        {
+            "properties": {
+                "addressSpace": {
+                    "addressPrefixes": [
+                        "10.2.0.0/24"
+                    ]
+                },
+                "subnets": [
+                    {
+                        "properties": {
+                            "addressPrefix": "10.2.0.0/24"
+                        },
+                        "name": "GatewaySubnet"
+                    }
+                ]
+            },
+            "name": "dev-vpn-vnet",
```

Comment:
dev-vpn-vnet is a newly added vnet which houses the VirtualNetworkGateway (VPN) client. Please refer to this diagram for how vnets are setup moving forward https://lucid.app/lucidchart/34183535-857f-45df-b68d-45772d0a27e3/edit?view_items=8caD-VenYEWG&invitationId=inv_282dd721-1a88-45af-8eba-f9c3938d3235#